spice/spice.py

```diff
@@ -400,6 +437,7 @@ async def stream_response(
         max_tokens: Optional[int] = None,
         response_format: Optional[ResponseFormat] = None,
         name: Optional[str] = None,
+        streaming_callback: Optional[Callable[[str], None]] = None,
```

Comment:
Why don't we have a validator / retries for stream_response?

Reply:
I think it would be better if we could do that but I'm not sure how without making the StreamingSpiceResponse a lot more complicated. It isn't aware of how to make the stream itself. Maybe you have a different approach in mind?

Reply:
No, I just read your other comment and I think it's ok.